sdk/template/ci.yml

```diff
@@ -56,10 +56,19 @@ parameters:
     displayName: 'azure-sdk-template-three'
     type: boolean
     default: true
+  # Switch to canary to test canary 1es branch. 1es template validation will set this parameter
```

Comment:
I can see how this will allow us to manually test in canary but does the 1ES team pass this parameter when they do test builds?

Reply:
@weshaggard as far as I could tell they have hardcoded release validation (https://dev.azure.com/azure-sdk/1ESPipelineTemplates/_git/1ESPipelineTemplates?path=/pipelines/validations/customer-validation-pipelines.yml&version=GBmain&line=12&lineEnd=13&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents). I'm going to start a thread to verify though.